<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRowDialog</class>
 <widget class="QDialog" name="addRowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="cityLabel">
     <property name="text">
      <string>City:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="cityLine"/>
   </item>
   <item>
    <widget class="QLabel" name="peopleLabel">
     <property name="text">
      <string>People:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="peopleBox">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="planetLabel">
     <property name="text">
      <string>Planet:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="planetBox">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="profitLabel">
     <property name="text">
      <string>Profit:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="profitBox">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="overallLabel">
     <property name="text">
      <string>Overall:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="overallBox">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="countryLabel">
     <property name="text">
      <string>Country:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="countryBox"/>
   </item>
   <item>
    <widget class="QLabel" name="contLabel">
     <property name="text">
      <string>Continent:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="contBox"/>
   </item>
   <item>
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
